### PR TITLE
PADV-1240 feat: add Manage instructors page

### PR DESCRIPTION
**Pull request by AuraAlba**
### Ticket
https://agile-jira.pearson.com/browse/PADV-1240

### Description
Add manage instructors page

### Changes made
Add Manage instructor page path
Add fetch status to add spinner in table and instructors list
Update tests

### Screenshots
Screenshot from 2024-05-22 15-49-38

### How to test
Clone the Institution Portal MFE
Run npm install.
Run npm run start
Go to http://localhost:1980/manage-instructors/:courseName/:className?classId=${classId} (same behavior as class detail page)

### CODE REVIEW

File: src/features/Instructors/ManageInstructorsPage/AssignSection.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      const instructorFilters = stateInstructors.filters;
+      dispatch(fetchInstructorsData(selectedInstitution.id, currentPage, instructorFilters));
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [currentPage, selectedInstitution, dispatch]);
+
+  return (
+    <Container size="xl" className="p-4 mt-3 instructors-content">
+      <h3>Assign / Add instructors</h3>
+      <InstructorsFilters isAssignSection resetPagination={resetPagination} />
```
Comment (01001110J) [inline src/features/Instructors/ManageInstructorsPage/AssignSection.jsx:74]: There's some design issues with this component:
- Here should have a magnifying glass icon 
missing icon in input

- The buttons should be next to input and not behind it

buttons in wrong posittion

File: src/assets/variables.scss [hunk 2]
```diff
@@ -1,6 +1,8 @@
+@import "assets/colors.scss";
 @import url("https://fonts.googleapis.com/css2?family=Open+Sans&display=swap");
 
 $font-family: "Open Sans", sans-serif;
 $account-menu-box-shadow: 2px 3px 8px 0 rgba(0, 0, 0, 0.5);
 
 $border-radius-1: 0.375rem;
+$info-100: $blue-50;
```
[the remark this answers is not in the document]
Comment (AuraAlba) [reply src/assets/variables.scss:8]: This is to override the variable of paragon

File: src/features/Instructors/AssignInstructors/index.jsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+// THIS FOLDER WILL BE REMOVED AFTER INTEGRATE ALL SECTIONS WITH NEW MANAGE INSTRUCTORS PAGE
```
[the remark this answers is not in the document]
Comment (AuraAlba) [reply src/features/Instructors/AssignInstructors/index.jsx:1]: I created the ticket for that

File: src/features/Instructors/ManageInstructorsPage/AssignSection.jsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  const clearSelectionRef = useRef(null);
+  const selectedInstitution = useSelector((state) => state.main.selectedInstitution);
+  const stateInstructors = useSelector((state) => state.instructors.table);
+  const [currentPage, setCurrentPage] = useState(initialPage);
+  const COLUMNS = useMemo(() => columns, []);
+  const isLoading = stateInstructors.status === RequestStatus.LOADING;
+
+  const selectColumn = {
+    id: 'selection',
+    Header: <></>, // eslint-disable-line react/jsx-no-useless-fragment
```
[the remark this answers is not in the document]
Comment (AuraAlba) [reply src/features/Instructors/ManageInstructorsPage/AssignSection.jsx:45]: This change give an error for the datatable

File: src/features/Instructors/ManageInstructorsPage/columns.jsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/prop-types, no-nested-ternary */
+import { differenceInHours, differenceInDays, differenceInWeeks } from 'date-fns';
```
[the remark this answers is not in the document]
Comment (AuraAlba) [reply src/features/Instructors/ManageInstructorsPage/columns.jsx:2]: The change of the label is not possible, we are limited with the table attributes and info with paragon. Color changed

File: src/features/Instructors/ManageInstructorsPage/index.jsx [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+      <Toast
+        onClose={() => setShowToast(false)}
+        show={showToast}
+      >
+        {toastMessage}
+      </Toast>
+      <Container size="xl" className="px-4 mt-3 manage-instructors-page">
+        <div className="d-flex justify-content-between mb-3 flex-column flex-sm-row">
+          <div className="d-flex align-items-center mb-3">
+            <Link to="/instructors" className="mr-3 link">
```
[the remark this answers is not in the document]
Comment (AuraAlba) [reply src/features/Instructors/ManageInstructorsPage/index.jsx:96]: The scope of this story is to not have a preceding page, just the page, I changed to courses but this is something you should keep in mind when developing the other stories related with this page.

File: src/features/Instructors/ManageInstructorsPage/index.jsx [hunk 7]
```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useRef, useState } from 'react';
```
[the remark this answers is not in the document]
Comment (AuraAlba) [reply src/features/Instructors/ManageInstructorsPage/index.jsx:1]: Is not part of the scope, we are gonna work on that for the next sprint

**Review by anfbermudezme — APPROVED** [review]

**Review by 01001110J — APPROVED** [review]

**Pull request merged**